connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -802,6 +826,16 @@ public String workerId() {
         return workerId;
     }
 
+    /**
+     * Returns whether this worker is configured to allow source connectors to create the topics
+     * that they use with custom configurations, if these topics don't already exist.
+     *
+     * @return true if topic creation by source connectors is allowed; false otherwise
+     */
+    public boolean isTopicCreationEnabled() {
+        return config.getBoolean(TOPIC_CREATION_ENABLE_CONFIG);
```

Comment:
Why not use the new method on WorkerConfig instead?
```suggestion
        return config.topicCreationEnable();
```

Reply:
All I can think of is that I added this one here first, thinking I might not need a similar method in the config. Fixed.